Tests/PayID/PayIDUtilsTests.swift

```diff
@@ -2,63 +2,67 @@ import XCTest
 import XpringKit
 
 class PayIDUtilsTest: XCTestCase {
-  func testParsePaymentPointerHostAndPath() {
-    // GIVEN a payment pointer with a host and a path.
-    let rawPaymentPointer = "$example.com/foo"
+  func testParseValidPayID() {
+    // GIVEN a Pay ID with a host and a path.
+    let host = "xpring.money"
+    let path = "georgewashington"
+    let rawPayID = "\(path)$\(host)"
```

Comment:
The variable names obscure this a little, so I just want to make sure we're on the same page about the format here.

`alice$xpring.money` => `https://xpring.money/alice`

`alice$xpring.money/example-path` => `https://xpring.money/example-path/alice`.

The format here is `{user}${domain.tld(/namespace-path)}`

Reply:
This is the first I'm hearing of supporting arbitrary paths. Given that no one has mentioned this, and I can't find it located anywhere else, I'm going to ignore this edge case for now. If/when we standardize on this, I'm happy to make sure it's supported with unit tests. 

I think that this will just work; but without a proper specification from product or engineering there's really no way to know.

Reply:
Ok, the executive decision is that if you want to namespace, use a subdomain, so you don't have to worry about the `path` stuff.